rfcs/055-genres-as-concepts/README.md

```diff
@@ -0,0 +1,138 @@
+# RFC 055: Genres as Concepts
+
+Genres (type/technique) should be treated in the same manner as Concepts.
+This includes introducing Concept pages for them that work in a similar manner to those
+for Agents (where works about and by the Agent are listed separately). A Concept
+page for a Genre should list works about and using that technique.
+
+## What is it like now?
+
+Currently, Genres behave a little like (compound) Subjects, in that it is 
+cracked into its constituent concepts, making a concepts list.
+
+However, unlike Subjects, the Genre-as-a-whole does not have its own name and identifier
+
+Genre is also not one of the types extracted from Works by the Concepts Aggregator.
+
+The constituent concepts that make up a Genre also behave in a similar manner
+to Subjects, where they are either a Concept ($a), or a more specific sort of Concept
+(e.g. Place, Period) depending on which subfield they come from.
+
+## Proposal - Catalogue Pipeline
+
+The primary Concept that is currently in a genre's concepts list
+(a Concept of type Concept, derived from the $a subfield) should now
+become a Concept of type Genre.
+
+### Genre as a whole
```

Comment:
[AFAICT, we only have three instances of a compound genre with an id in $0](https://c783b93d8b0b4b11900b5793cb2a1865.eu-west-1.aws.found.io:9243/s/sierra/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:now-15m,to:now))&_a=(columns:!(),filters:!(('$state':(store:appState),meta:(alias:!n,disabled:!f,index:d5ea4b8c-f58d-409c-8fb7-8a9973ea67f7,key:varField.marcTag,negate:!f,params:(query:'655'),type:phrase),query:(match_phrase:(varField.marcTag:'655'))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:d5ea4b8c-f58d-409c-8fb7-8a9973ea67f7,key:varField.subfields.tag,negate:!f,params:!(z,y,x,v),type:phrases),query:(bool:(minimum_should_match:1,should:!((match_phrase:(varField.subfields.tag:z)),(match_phrase:(varField.subfields.tag:y)),(match_phrase:(varField.subfields.tag:x)),(match_phrase:(varField.subfields.tag:v)))))),('$state':(store:appState),meta:(alias:!n,disabled:!f,index:d5ea4b8c-f58d-409c-8fb7-8a9973ea67f7,key:varField.subfields.tag,negate:!f,params:(query:'0'),type:phrase),query:(match_phrase:(varField.subfields.tag:'0')))),index:d5ea4b8c-f58d-409c-8fb7-8a9973ea67f7,interval:auto,query:(language:kuery,query:''),sort:!()))


All of them are [exhibition catalogues in the GND scheme](https://portal.dnb.de/opac.htm?method=simpleSearch&cqlMode=true&query=nid%3D4135467-9). Notably, the identifier refers to all Austellungskataloge, the other subfields embellish this, rather than form part of it.